javascript-data-model/order-history.js

```diff
@@ -1,5 +1,5 @@
 var orderHistory = [
-  order1 = [
+  [
```

Comment:
This is still incorrect, you don't need to wrap each individual order object in an array, there is already an array to hold all orders in the `orderHistory` variable.

Reply:
![Screen Shot 2021-01-28 at 4 30 50 PM](https://user-images.githubusercontent.com/69738204/106201326-4cfb9480-6186-11eb-8866-2000207f7e66.png)
![Screen Shot 2021-01-28 at 4 30 56 PM](https://user-images.githubusercontent.com/69738204/106201341-4e2cc180-6186-11eb-9321-a100e2e396cb.png)